test/unit/streaming-client.spec.ts

```diff
@@ -38,12 +38,14 @@ describe('streaming-client', () => {
                 `?access_token=${token}` +
                 `&content_type=${audioConfig.getContentTypeString()}` +
                 `&user_agent=${encodeURIComponent(`RevAi-NodeSDK/${sdkVersion}`)}` +
-                `&metadata=${encodeURIComponent(config.metadata)}`
+                `&metadata=${encodeURIComponent(config.metadata)}` +
+                `&custom_vocabulary_id=${encodeURIComponent(config.customVocabularyID)}` +
+                `&filter_profanity=${encodeURIComponent(config.filterProfanity)}`
```

Comment:
In streaming-client.ts line 78 you had a .toString() after the filterProfanity property, I think you should keep this line consistent with that line (either both with .toString() or both without)

Reply:
Addressed this in the streaming client!

Reply:
Why wouldn't this work in the streaming client, but it works in this test file?

Reply:
Discussed and addressed satisfactorily.